core/foundation/src/TClassEdit.cxx

```diff
@@ -771,7 +771,10 @@ static bool IsDefElement(const char *elementName, const char* defaultElementName
       size_t end = findNameEnd(c,pos);
 
       std::string keypart;
-      TClassEdit::GetNormalizedName(keypart,std::string_view(c.c_str()+pos,end-pos));
+      if (pos != end) {
+         // ROOT-11000: elementName = "std::less<>"
```

Comment:
The comment is "misplaced".  Visually it looks like it explains the "next" line while it actually explains the previous line.
```suggestion
      if (pos != end) {  // ROOT-11000: elementName = "std::less<>"
```